src/Action/BaseAction.php

```diff
@@ -26,6 +26,15 @@
     protected $_responding = false;
 
     /**
+     * Default configuration
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'modelFactory' => 'Table',
```

Comment:
It will still cause problem for custom actions people have. You can instead check if the key is set in ProxyTrait and use if available.